seatunnel-translation/seatunnel-translation-spark/pom.xml

```diff
@@ -24,6 +24,11 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>seatunnel-translation-spark</artifactId>
+    <packaging>pom</packaging>
+    <modules>
+        <module>seatunnel-translation-spark-2.4</module>
+        <module>seatunnel-translation-spark-common</module>
+    </modules>
 
     <properties>
         <spark.version>2.4.0</spark.version>
```

Comment:
The `<properties>` and `<dependencies>` blocks are useless because of `<packaging>pom</packaging>`, please delete them.

Reply:
This module is the parent module of multiple spark version translation modules, so you cannot manage `spark.version` in this module.